scripts/find-active-suppliers-since-a-date.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+"""
+This script was needed to find all active registered suppliers, since a given date.
+
+The script parses a date, provided by the user (if not default date of today is provided).
+This date is then compared against the Suppliers list, where active is true.
+If the date is more recent than the date the user provided, the record is captured and outputted.
+A count is taken and also provided.
+
+Usage:
+    scripts/find-active-suppliers-since-a-date.py <stage> [options]
+
+Options:
+    --date=YYYY-MM-DD       Limit the number of suppliers to find [default: <today's date>]
```

Comment:
If I try to use the default I get:

```
$ scripts/find-active-suppliers-since-a-date.py preview
ValueError: time data "<today's date>" does not match format '%Y-%m-%d'
```